
---

uint8 NO_OVERTAKE=0
uint8 OVERTAKING=1
uint8 OVERTAKE_QUEUED=2
uint8 status